frontend/src/pages/store/[store].svelte

```diff
@@ -514,8 +515,28 @@
 								maximumFractionDigits: 2,
 							})}
 						</p>
+						{#if item.quantity != null}
+							<p class="text-2xl font-bold">
+								{#if item.quantity == 0}
+									<!-- This is not possible now but it is here if we need it -->
+									<span class="text-red-400"
+										>OUT OF STOCK</span
+									>
+								{:else if item.quantity <= LOW_STOCK}
+									<span class="text-yellow-400"
+										>LOW STOCK</span
+									>
+								{:else}
+									<span class="text-green-400">IN STOCK</span>
+								{/if}
+								<span class="text-gray-400">&nbsp;-&nbsp;</span>
+								<span class="italic"
+									>{item.quantity.toLocaleString()}</span
+								>
+							</p>
+						{/if}
 						{#if item.description}
-							<p class="text-xl mt-2 whitespace-pre-wrap">
+							<p class="text-xl mt-2 whitespace-pre-wrap italic">
```

Comment:
Why did you make this italic?

Reply:
Sometimes italics are used for quotes and the description of somewhat of a quote because you are describing the item

I also thought it looked a bit better by setting it apart from the other things already there, like price, name, etc.

Reply:
Alright. In that case don't make the quantities italic like I suggested.